sentry-kotlin-extensions/src/main/java/io/sentry/kotlin/SentryCoroutineExceptionHandler.kt

```diff
@@ -0,0 +1,29 @@
+package io.sentry.kotlin
+
+import io.sentry.HubAdapter
+import io.sentry.IHub
+import io.sentry.SentryEvent
+import io.sentry.exception.ExceptionMechanismException
+import io.sentry.protocol.Mechanism
+import kotlinx.coroutines.CoroutineExceptionHandler
+import org.jetbrains.annotations.ApiStatus
+import kotlin.coroutines.AbstractCoroutineContextElement
+import kotlin.coroutines.CoroutineContext
+
+/**
+ * Sentry exception handler element for [CoroutineExceptionHandler].
+ */
+@ApiStatus.Experimental
+public class SentryCoroutineExceptionHandler(private val hub: IHub = HubAdapter.getInstance()) : AbstractCoroutineContextElement(CoroutineExceptionHandler), CoroutineExceptionHandler {
```

Comment:
Should we make it `open` for inheritance? If I have my custom exception handler I'd want to have mine on top of Sentry's, or we advise delegating?

Reply:
Probably a good idea, other integrations had to make it open because of that.